src/main/markdown/lifeofanissue.md

```diff
@@ -12,9 +12,9 @@ One of the best ways you can help us improve GWT is to let us know about any pro
 
 Here's how to report issues:
 
-1.  [Search for your issue](http://code.google.com/p/google-web-toolkit/issues/advsearch) to see if anyone has already reported it.
+1.  [Search for your issue](https://github.com/gwtproject/gwt/issues) to see if anyone has already reported it.
```

Comment:
Maybe we could provide an initial "filter" that removes the default `is:open` so people don't miss closed issues?

That would be `https://github.com/gwtproject/gwt/issues?q=is%3Aissue`